apify-docs-theme/src/theme/custom.css

```diff
@@ -606,6 +606,10 @@ article .card h2 {
     user-select: none;
 }
 
+.tsd-flag-group {
```

Comment:
this doesnt look right, we already have margin on the group, its missing on `tsd-flag`

<img width="386" alt="image" src="https://github.com/user-attachments/assets/2a49eaf3-984d-4bee-98de-b6a3e9b8e93f">

<img width="209" alt="image" src="https://github.com/user-attachments/assets/ec8823ee-48bb-4e04-86c3-5da9c59a8700">

for `tsd-flag` we want just `margin-right`, it has a negative top margin and you would override it this way

Reply:
hmm the screen above is parameters, but the one in OP is on higher level (method name). this will be probably more tricky, you will need to target this better most likely.

try to debug this directly on the SDK docs so you can see how things end up visually. you can run them separately.

Reply:
Ahh.. I see.
Should be fixed now ✅ 
![CleanShot 2024-11-18 at 13 00 30@2x](https://github.com/user-attachments/assets/22295fd9-8896-444c-a493-7b1697801279)